docs/examples/unenforced_constraints.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel, Field, PositiveInt
+
+try:
+    # this won't works since PositiveInt takes precedence over the
+    # constraints defined in Field meaning they're ignored
+    class Model(BaseModel):
+        foo: PositiveInt = Field(..., lt=10)
+except ValueError as e:
+    print(e)
+
+# but you can set the schema attribute directly:
+# (Note: here exclusiveMaximum will not be enforce)
+class Model(BaseModel):
+    foo: PositiveInt = Field(..., exclusiveMaximum=10)
+
+print(Model.schema())
```

Comment:
```suggestion
print(Model.schema())

# if you find yourself needing this, an alternative is to declare the constraints in Field:
class Model(BaseModel):
    # Here both constraints will be applied and the schema will be generated correctly
    foo: int = Field(..., gt=0, lt=10)

print(Model.schema())
```

Reply:
need to respect line length now :-)

done.